images/macos/macos-11-Readme.md

```diff
@@ -0,0 +1,298 @@
+| Announcements |
+|-|
+| [macOS 11.0 pools will be transited to private preview.](https://github.com/actions/virtual-environments/issues/2486) |
```

Comment:
Should we fix this announcement as well? 😃

Reply:
I think we should leave it as-is, do not see any sense in changing it